examples/GLTextItem.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+"""
+Simple examples demonstrating the use of GLTextItem.
+
+"""
+
+from pyqtgraph.Qt import QtCore, QtGui, mkQApp
+import pyqtgraph.opengl as gl
+
+app = mkQApp("GLTextItem Example")
+
+gvw = gl.GLViewWidget()
+gvw.show()
+gvw.setWindowTitle('pyqtgraph example: GLTextItem')
+
+txtitem1 = gl.GLTextItem(pos=(0.0, 0.0, 0.0), text='text1')
+gvw.addItem(txtitem1)
+
+txtitem2 = gl.GLTextItem()
+txtitem2.setData(pos=(1.0, -1.0, 2.0), color=(0.5, 1.0, 0.5, 1.0), text='text2')
+gvw.addItem(txtitem2)
+
+if __name__ == '__main__':
+  mkQApp().exec_()
```

Comment:
we _just_ changed this (like I merged the PR with this diff a few minutes ago).  

Can you change this to 

```python
pg.exec()
```

This gives Qt 6.1 compatibility without emitting deprecation warnings (looks like you'll have to `import pyqtgraph as pg` at the top too

Reply:
I fixed in [
b98244e](https://github.com/pyqtgraph/pyqtgraph/pull/1776/commits/b98244e7d34ce368d6d49af76545daf2d4f0ade1).